### ISSUE TITLE
Create a NuGet package

### ISSUE THREAD

**Issue by leo-schick** [libfintx/libfintx#3]
This is a follow up for https://github.com/torstenklement/libfintx/issues/116

Goal is to provide a NuGet package with:
* Framework support for .NET 8, .NET 6 and .NET Standard 2 (if applicable)
* Add a strong name to avoid duplicated loading when using dynamic loading
* deploy it via a continuous delivery GitHub action

**Comment by yunits** [comment]
+1 would be nice

**Comment by leo-schick** [comment]
Current status: The package can be build, but cannot be published to NuGet, because there is an unlisted package with the same name. I send in a request to the NuGet Team to claim ownership, so that I could publish a new NuGet package version and to list the package again, so that people can find it via the NuGet package search.

**Comment by torstenklement** [comment]
@leo-schick now I am fully available. I saw you own the package now. Can you delete this?

**Comment by leo-schick** [comment]
@torstenklement Yes, yesterday I got the ownership from the NuGet Team. We could now create a NuGet Organization for libfintx to manage it. Do you want to create and invite me, so I can transfer ownership?

I am not sure if I understand you right, but I think we should not delete this issue. There are still some tasks undone:
* Add a strong name to avoid duplicated loading when using dynamic loading
* deploy it via a continuous delivery GitHub action
see above.

**Comment by leo-schick** [comment]
**Status update:**

- I prepared the libfintx.FinTS part of the package to be deployed as NuGet package (adding description, tags, icons etc.)
- I build a continuous deployment pipeline which will deploy it automatically on version upgrade, see https://github.com/libfintx/libfintx/pull/8
- I tried to upload some basis packages to NuGet, but discovered, that some other packages where already created by [FinanceKey](https://github.com/financeKey/) in the past which are unlisted. I send her/him a message via NuGet and requested ownership transfer for these packages:
  - [libfintx.Globals](https://www.nuget.org/packages/libfintx.Globals)
  - [libfintx.FinTS](https://www.nuget.org/packages/libfintx.FinTS)

**Comment by leo-schick** [comment]
*Status update:*
* Since I did not get any reply from FinanceKey, I requested ownership for the packages on NuGet
* The CI/CD pipeline is now merged and a new NuGet package is automatically deployed for packages when the version number is increased. Currently only active for package libfintx.Logger and libfintx.Sepa

**Comment by leo-schick** [comment]
> Hello, 
>
> We have sent an email to the owners of the package. 
>
> If they do not respond within 30 days, we will add you as a co-owner of the package. 
> 
> Best regards, 
> 
> NuGet Admin 

🎉

**Comment by leo-schick** [comment]
The following packages are now released on NuGet:

- [libfintx.FinTS](https://www.nuget.org/packages/libfintx.FinTS) - Version 1.1.0
- [libfintx.Globals](https://www.nuget.org/packages/libfintx.Globals) - Version 1.1.0
- [libfintx.Sepa](https://www.nuget.org/packages/libfintx.Sepa/) - Version 1.0.0
- [libfintx.Logger](https://www.nuget.org/packages/libfintx.Logger) - Version 1.0.0

It is now included in the continuous distribution workflow in GitHub. A new version is published everytime the version of a package is increased.
The publishing of the NuGet packages for EBICS is followed up in #10.
